mod.ts

```diff
@@ -6,14 +6,13 @@ if (CAPI_MARKER in globalThis) {
 
 export * as $ from "./deps/scale.ts"
 export { BitSequence } from "./deps/scale.ts"
-export * from "./server/client/mod.ts"
+export * from "./server/client/detectConnect.ts"
 
 // moderate --exclude main.ts nets.ts util server
 
 export * from "./crypto/mod.ts"
 export * from "./fluent/mod.ts"
 export * from "./frame_metadata/mod.ts"
-export * from "./nets/mod.ts"
 export * from "./rpc/mod.ts"
 export * from "./rune/mod.ts"
 export * from "./scale_info/mod.ts"
```

Comment:
`./nets/mod.ts` and `./server/client/mod.ts` have functions that close over the `devUserPublicKeysData`.

I believe that the main `esm` entry should be optimized for the browser.
If so, `./nets/mod.ts` is only needed for the `nets.ts` file (which runs in node/deno) and the `import` could import `capi/nets` instead of `capi`.

Then, from `./server/client/mod.ts` only `detectConnect.ts` is needed for the browser by the target chain `connection.js`.

@harrysolovay @tjjfvi WDYT?